docs/sources/style-guide/style-conventions/index.md

```diff
@@ -130,54 +131,10 @@ A tip describes a more efficient or alternate way of doing something. Tips are r
 
 A caution warns the user to proceed with caution. A caution emphasizes a course of action's potential downsides.
 
-The markdown syntax for a caution is `> **Caution**` and will appear in the published output as shown in the following example:
-
 > **Caution:** By disabling authentication requirements, anyone can access your Grafana instance. There is a considerable security risk associated with this.
 
 ### Warnings
 
-A warning informs the user not to do something.  Warnings are usually reserved for actions that, if performed, could cause harm to hardware, software, or data.  
-
-The Markdown syntax for a warning is `> **Warning:**`. The warning appears in the published output as follows:
+A warning informs the user not to do something For example:
 
 > **Warning:** You cannot back up your dashboards in Grafana. You might not be able to recover a dashboard if it is deleted.
```

Comment:
```suggestion
> **Warning:** Do not back up your dashboards in Grafana. You might not be able to recover a dashboard if it is deleted.
```
It might be possible, but iladvised.

Reply:
This warning seems to contradict itself somehow.

Reply:
What am I supposed to do instead?